### DISCUSSION TITLE
Step back function for playground

### DISCUSSION THREAD

**Discussion by degatchi** [smlxl/evm.codes#201] [Ideas]
Pls add a step back function/the opposite direct to `Step into` for playground mode

**Comment by peroket** [comment]
Can you tell us what you want to do? Not the implementation you want, but the functionality you are missing. What would stepping back allow you to do in your use case?
Thanks :-)

**Reply by degatchi** [reply]
When im skipping through each step and miss something i have to replay the entire tx. It's a big time inconvenience.

**Reply by peroket** [reply]
And the breakpoints could not help you with that?

**Reply by degatchi** [reply]
Breakpoints?

**Reply by peroket** [reply]
When you load code, and hover your mouse above it, you see a red dot appearing. If you click on it, you can then let the code run, and it will stop when it reaches that instruction.

**Reply by degatchi** [reply]
Oh nah that doesn't help. Need to be able to reverse the steps.

**Reply by peroket** [reply]
Thank you for your feedback.
I'm not hiding that it's probably not going to come soon, as this is not an easy functionality to add, and we are focused on other things, but we will keep it in mind.
Also feel free to give it a go yourself :-)
